codyn/codyn_examples.Rmd

```diff
@@ -0,0 +1,214 @@
+  #remove bout comparisons that cross years
+  filter(year(bout) == year(bout2))
+```
+
+Mammals
+```{r}
+#sites that meet sampling frequency criteria 
+mam_inc_sites <- readRDS(file="data/Mammals/NEONsites_tempreplic_4yrs.rds") %>% 
+  rename(siteID = SiteID)
+
+mam_inter <- readRDS("~/Documents/Projects/temporalNEON/data/Mammals/DATA_mammals_abs-abund-inter.rds") %>%
+  pivot_longer(-c(siteID, year), names_to = "scientificName", values_to = "abundance") %>%
+  filter(siteID %in% mam_inc_sites$siteID)
+
+mam_intra <- readRDS("~/Documents/Projects/temporalNEON/data/Mammals/DATA_mammals_abs-abund-intra.rds") %>%
+  pivot_longer(-c(siteID, year, month, bout), names_to = "scientificName", values_to = "abundance") %>%
+  filter(siteID %in% mam_inc_sites$siteID)
+
+
+mam_change_inter <- RAC_change(mam_inter, 
+                             time.var = "year", species.var = "scientificName",
+                             abundance.var = "abundance", replicate.var = "siteID")
+
+mam_change_intra <- RAC_change(mam_intra, 
+                             time.var = "bout", species.var = "scientificName",
+                             abundance.var = "abundance", replicate.var = "siteID") %>%
+  #remove bout comparisons that cross years
+  filter(substr(bout, 1, 4) == substr(bout2, 1, 4))
+```
+
+Fish
+```{r}
+fish_inter <- readRDS("~/Documents/Projects/temporalNEON/data/fish/fsh_BAT_inter.rds") %>%
+  pivot_longer(-c(siteID, year), names_to = "scientificName", values_to = "abundance")
+
+fish_intra <- readRDS("~/Documents/Projects/temporalNEON/data/fish/fsh_BAT_intra.rds") %>%
+  pivot_longer(-c(siteID, year, month, bout), names_to = "scientificName", values_to = "abundance")
+
+#sites that match sampling criteria
+fish_inc_sites <- fish_intra %>% 
+  group_by(siteID) %>% 
+  filter(n_distinct(bout) > 2, n_distinct(year) > 3)
+
+fish_change_inter <- RAC_change(fish_inter %>% filter(siteID %in% fish_inc_sites$siteID), 
+                             time.var = "year", species.var = "scientificName",
+                             abundance.var = "abundance", replicate.var = "siteID")
+
+fish_change_intra <- RAC_change(fish_intra, 
+                             time.var = "bout", species.var = "scientificName",
+                             abundance.var = "abundance", replicate.var = "siteID") %>%
+  #remove bout comparisons that cross years
+  filter(substr(bout, 1, 4) == substr(bout2, 1, 4))
+```
+
+Inverts
+```{r}
+invert_inter <- read_csv("~/Documents/Projects/temporalNEON/data/inverts/inverts_table_year_BAT.csv") %>%
+  pivot_longer(-c(siteID, year), names_to = "scientificName", values_to = "abundance")
+
+invert_intra <- read_csv("~/Documents/Projects/temporalNEON/data/inverts/inverts_table_bout_BAT.csv") %>%
+  group_by(siteID, bout, year, month) %>%
+  summarise_all(funs(sum)) %>%
+  pivot_longer(-c(siteID, year, month, bout), names_to = "scientificName", values_to = "abundance")
+
+invert_inc_sites <- fish_intra %>% 
```

Comment:
invert_intra  
and not 
fish_intra

??

Reply:
Whoops, good catch! Thanks @mperezrocha83, fixed now and figures in the doc updated.